src/components/_settings.scss

```diff
@@ -375,6 +375,7 @@ $bgc-range-datepicker-selected: $c-primary !default;
 
 // Collapsible
 $bgc-collapsible-basic: $c-gray-lightest !default;
+$bgc-expanded-collapsible-basic: $c-gray-lightest !default;
```

Comment:
what do you think about `$bgc-collapsible-basic-expanded` ?
looks like more consistent naming to me.

Reply:
Done! What you've suggested seems to be the current convention 👍

Reply:
awesome, thank you!